screens/Surveys.js

```diff
@@ -41,5 +66,9 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     alignItems: 'center',
     backgroundColor: '#644B62'
-  }
+  },
+  buttonContainer: {
+    flex: 1,
+    alignContent: 'flex-start',
+  },
```

Comment:
Could you revert all the changes from the surveys file to the way it currently is on develop or do you actually need the code in this file?